assets/src/js/admin/admin-scripts.js

```diff
@@ -60,37 +60,40 @@ var give_setting_edit = false;
 	var give_dismiss_notice = function () {
 		$('body').on('click', 'button.notice-dismiss', function () {
 			if ('give-invalid-license' !== jQuery(this).closest('div.give-notice').data('notice-id')) {
-				give_remove_give_message();
+				give_remove_give_message( jQuery(this).closest('div.give-notice').attr( 'id' ) );
 			}
 		});
 	};
 
-	/**
-	 * Remove give-message parameter from URL.
-	 *
-	 * @since 1.8.14
-	 */
-	var give_remove_give_message = function () {
-		var parameter = 'give-message',
-			url = document.location.href,
-			urlparts = url.split('?');
-
-		if (urlparts.length >= 2) {
-			var urlBase = urlparts.shift();
-			var queryString = urlparts.join('?');
-
-			var prefix = encodeURIComponent(parameter) + '=';
-			var pars = queryString.split(/[&;]/g);
-			for (var i = pars.length; i-- > 0;) {
-				if (pars[i].lastIndexOf(prefix, 0) !== -1) {
-					pars.splice(i, 1);
-				}
-			}
-			url = urlBase + '?' + pars.join('&');
-			window.history.pushState('', document.title, url); // added this line to push the new url directly to url bar .
-		}
-		return url;
-	};
+    /**
+     * Remove give-message parameter from URL.
+     *
+     * @since 1.8.14
+     */
+    var give_remove_give_message = function (key) {
```

Comment:
@raftaar1191 Improve code formatting. I can see space instead of tab.

![image](https://user-images.githubusercontent.com/1784821/40595916-324482c4-6255-11e8-9672-7b8b982fb68e.png)

Reply:
@ravinderk PR updated with WP coding standard